Dockerfile.debian

```diff
@@ -0,0 +1,52 @@
+# Neolink Docker image build scripts
+# Copyright (c) 2020 George Hilliard
+# SPDX-License-Identifier: AGPL-3.0-only
+
+FROM docker.io/rust:buster AS build
+MAINTAINER thirtythreeforty@gmail.com
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    musl-dev gcc \
```

Comment:
Probably don't need `musl-dev` on a buster.

Reply:
Without it, it didn't work.